<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnLote</class>
 <widget class="QWidget" name="vtnLote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta Lote</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbLote">
     <property name="title">
      <string>Lote</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Código</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineCodigo">
          <property name="statusTip">
           <string>Ingrese el número de Lote del Producto (solo letras y números sin espacios)</string>
          </property>
          <property name="accessibleDescription">
           <string>codLote</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBuscarLote">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fecha de Vencimiento</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDateEdit" name="dateFechVenc">
          <property name="statusTip">
           <string>Ingrese la fecha de vencimiento del lote</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCantidad">
        <property name="text">
         <string>Cantidad</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QSpinBox" name="spinCantidad">
          <property name="statusTip">
           <string>Ingrese la cantidad del Producto para dicho lote</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTableWidget" name="tablaLote">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Código</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fecha de Vencimiento</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnActualizarLote">
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProducto">
     <property name="title">
      <string>Producto</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>* Código de Barra</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineCod_Barra">
          <property name="statusTip">
           <string>Ingrese código de barra del producto (exactamente 9 números)</string>
          </property>
          <property name="accessibleDescription">
           <string>codigo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBuscarProd">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tablaProducto">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Código de Barra</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Medicamento</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Presentación</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Importe</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnActualizarProd">
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineCodigo</tabstop>
  <tabstop>btnBuscarLote</tabstop>
  <tabstop>dateFechVenc</tabstop>
  <tabstop>spinCantidad</tabstop>
  <tabstop>tablaLote</tabstop>
  <tabstop>btnActualizarLote</tabstop>
  <tabstop>lineCod_Barra</tabstop>
  <tabstop>btnBuscarProd</tabstop>
  <tabstop>tablaProducto</tabstop>
  <tabstop>btnActualizarProd</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
